### PR TITLE
saving get.

### PR DESCRIPTION
**Pull request by meyerj19**

### CODE REVIEW

File: src/components/App/App.jsx [hunk 1]
```diff
@@ -1,10 +1,33 @@
...
+import { useEffect, useState } from 'react';
 
 import Header from '../Header/Header.jsx'
 import './App.css';
 
 
 function App() {
+
+    useEffect(() => {
+        fetchCreatures();
```
Comment (NateMacLaurin) [inline src/components/App/App.jsx]: fetchCreatures() should be fetchList()

Comment (travisjhuss) [reply src/components/App/App.jsx]: I am writing a comment

**Comment by meyerj19** [comment]
changed fetchCreature to fetchList

File: src/components/App/App.jsx (hunk 1 above)
Comment (meyerj19) [reply src/components/App/App.jsx]: made change

**Pull request merged**